src/utils/validateIdentifierInputs.ts

```diff
@@ -1,5 +1,6 @@
 import { IdentifierInput } from "../types";
 import { fullPhoneNumberValidator } from "./validatePhoneNumbers";
+import { ERROR_MESSAGE } from "../context/alert";
```

Comment:
Hmm, actually instead of snake case, should we use camel case? ahaha

Reply:
Eh constants is snake case right?

Reply:
hmm, ok my bad haha, I went to read up further. I got confused initially. Let's not change it!

Based on [this](https://stackoverflow.com/questions/40291766/naming-convention-for-const-object-keys-in-es6), I think you're right, dont need to change haha